corpus/source-file-structure.txt

```diff
@@ -110,6 +110,24 @@ namespace A {
         (qualified_name (identifier) (identifier))
         (declaration_list)))))
 
+=====================================
+File scoped namespaces
+=====================================
+
+namespace A;
+
+class B {
+}
+
+---
+
+(compilation_unit
+  (file_scoped_namespace_declaration
+    (identifier)
+  (class_declaration
```

Comment:
just a nit, here it should be indented to show that the class is nested under the namespace declaration.